cypress/e2e/marc-authority/marc-authority-edit-verify-highlighted-heading-and-third-pane.cy.js

```diff
@@ -0,0 +1,148 @@
+import getRandomPostfix from '../../support/utils/stringTools';
+import { DevTeams, Permissions, TestTypes } from '../../support/dictionary';
+import TopMenu from '../../support/fragments/topMenu';
+import Users from '../../support/fragments/users/users';
+import MarcAuthorities from '../../support/fragments/marcAuthority/marcAuthorities';
+import MarcAuthority from '../../support/fragments/marcAuthority/marcAuthority';
+import DataImport from '../../support/fragments/data_import/dataImport';
+import JobProfiles from '../../support/fragments/data_import/job_profiles/jobProfiles';
+import Logs from '../../support/fragments/data_import/logs/logs';
+
+describe('Edit Authority record', () => {
+  const testData = {
+    authority: {
+      title: 'Twain, Mark, 1835-1910. Adventures of Huckleberry Finn',
+      searchInput: 'Twain, Mark',
+      searchOption: 'Name-title',
+    },
+    authority2: {
+      searchInput: 'Gulf Stream',
+      searchOption: 'Geographic name',
+    },
+    editedFields: [
+      {
+        tag: '100',
+        content: 'Twain, Mark, 1835-1910. Adventures of Huckleberry Finn - edited',
+        secondContent: 'Twain, Mark, 1835-1910. Adventures of Huckleberry Finn - edited twice',
+      },
+      { tag: '370', content: 'edited 370' },
+    ],
+    editedGeographicNameField: {
+      tag: '151',
+      content: 'edited 151',
+    },
+  };
+  const authorityPostfix = '?authRefType=Authorized&heading';
+  const jobProfileToRun = 'Default - Create SRS MARC Authority';
+  const marcFiles = [
+    {
+      marc: 'marcAuthFileC350911.mrc',
+      fileName: `testMarcFile.${getRandomPostfix()}.mrc`,
+      numOfRecords: 1,
+    },
+    {
+      marc: 'marcFileForC350946.mrc',
+      fileName: `testMarcFile.${getRandomPostfix()}.mrc`,
+      numOfRecords: 2,
+    },
+  ];
+  const createdAuthorityIDs = [];
+
+  before('Create test data', () => {
```

Comment:
Hi Tetiana, thanks for the review! When I tried to upload files through admin account, I often got the message "File upload already in progress". My logic was that  some other test might be uploading files parallelly and they disrupt each other, so it is better to upload files with an independent user account, to avoid this issue (and it worked).

Reply:
Fixed